moveit_ros/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_ros_integration_tests)
```

Comment:
These are actually ROS-related integration tests. That's why I decided to put them into moveit_ros originally.